src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -28,9 +29,8 @@
 
 
 class MozLintIssue(Issue):
-    def __init__(self, repo_dir, repo_path, column, level, lineno, linter, message, rule, **kwargs):
+    def __init__(self, repo_path, column, level, lineno, linter, message, rule, **kwargs):
```

Comment:
What's repo_path?

Reply:
`repo_path` is the path of the file for this issue, in mozilla-central

Reply:
Maybe `path` would be clearer, to avoid confusion